### PR TITLE
Add CLI doc guidelines to Community section

### PR DESCRIPTION
**Pull request by chenette**
This commit includes a new "Doc Guidelines" section with all planned guidelines (to minimize merge conflicts). Some links will be broken until the other guideline PRs have been merged.

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: community/doc_guidelines/clis.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# CLI Help Guidelines
+
```
Comment (agunde406) [inline community/doc_guidelines/clis.md:2]: Missing copyright

File: community/doc_guidelines/clis.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  - Use the `splinter keygen` command to generate user keys.
+
+  - Use the `splinter keygen` command to generate user keys.
+    <br><br>
+
+* If you choose to say "command line interface", always use the full term
+  (don't omit "interface").
+
+* Avoid starting a sentence with a word that is normally lowercase. For example, 
+  Change "splinterd" to "Splinter daemon".
```
Comment (agunde406) [inline community/doc_guidelines/clis.md:26]: Change -> change

File: community/doc_guidelines/clis.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* Do not add a period at the end of the description.
+
+## FLAGS and OPTIONS Sections
+
+* Note this terminology:
+  - A _flag_ occurs alone (without an argument). Do not use the term "switch".
+  - An _option_ has at least one _argument_.
+  - For equivalent items in the configuration file, use the term "configuration
+    setting" or "setting".
+
```
Comment (agunde406) [inline community/doc_guidelines/clis.md:49]: adding `<br><br>` here would provide better formatting of this list

File: community/doc_guidelines/clis.md (hunk 1 above)
Comment (chenette) [reply community/doc_guidelines/clis.md:2]: Added.

File: community/doc_guidelines/clis.md (hunk 2 above)
Comment (chenette) [reply community/doc_guidelines/clis.md:26]: Fixed.

File: community/doc_guidelines/clis.md (hunk 3 above)
Comment (chenette) [reply community/doc_guidelines/clis.md:49]: Done.

**Comment by ltseeley** [comment]
Commit was cherry-picked in #131, closing

**Pull request closed without merging**